extensions/objects/templates/image-with-regions-editor.component.ts

```diff
@@ -29,41 +29,56 @@ import { CustomSchema } from 'services/schema-default-value.service';
 import { UtilsService } from 'services/utils.service';
 import { ImageWithRegionsResetConfirmationModalComponent } from './image-with-regions-reset-confirmation.component';
```

Comment:
Throwing errors in this particular file at instances where we're typecasting values with number will have different impact on the UI. There are console errors, e2e errors and different frontend failures on adding if checks or throwing errors at places where we've typecasted the value.

Reply:
Basically calculations we're doing here in this file would be affected on throwing errors or using if checks. Example failures can be found here https://github.com/oppia/oppia/runs/3228768338